contract_api/registry.py

```diff
@@ -135,7 +135,7 @@ def _search_query_data(self, sub_qry, sort_by, order_by, offset, limit, filter_q
             print("qry_part::", qry_part)
             sort_by = sort_by.replace("org_id", "M.org_id")
             services = self.repo.execute(
-                "SELECT M.*,O.organization_name,O.org_assets_url FROM service_endpoint E, service_metadata M, service S "
+                "SELECT DISTINCT M.*,O.organization_name,O.org_assets_url FROM service_endpoint E, service_metadata M, service S "
```

Comment:
If a service has multiple different endpoints you will still get multiple rows for the above query. Does that not cause the problem again?

Reply:
1. If a service has multiple different endpoints, you will not get duplicate records
2. Because `M.*,O.organization_name,O.org_assets_url` is going to be unique.
3. I have tested this query for service_id with multiple endpoints.